packages/pilot/src/containers/Balance/index.js

```diff
@@ -458,8 +469,11 @@ class Balance extends Component {
                       icon={<IconCalendar width={16} height={16} />}
                       limits={dateLimits}
                       onChange={this.handleDatesChange}
-                      presets={datePresets}
-                      strings={getDateLabels(this.props.t)}
+                      onPresetChange={this.handlePresetChange}
+                      presets={this.localizedPresets}
+                      selectedPreset="last-7"
+                      strings={this.dateLabels}
+                      showCalendar={this.state.showDateInputCalendar}
                       value={this.state.dates}
```

Comment:
```suggestion
                      dates={this.state.dates}
```

essa modificacação resolve o problema reportado aqui https://github.com/pagarme/pilot/pull/1111#issuecomment-471108100

Reply:
Boa, resolvido!